get_all_data/common_field_transforms.py

```diff
@@ -0,0 +1,40 @@
+import dateutil.parser
+import datetime
+
+def convert_to_header(field):
```

Comment:
I think it would be a great idea to provide some common formatting and transformation functions that people can leverage.